main.tf

```diff
@@ -2,12 +2,12 @@
 ## defaults
 ################################################################################
 terraform {
-  required_version = "~> 1.3, < 2.0.0"
+  required_version = "~> 1.4"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 4.0"
+      version = "~> 4.65"
```

Comment:
aws should be 5.0 
and terraform required_version was correct , please dont change this 
cc @vijay-stephen  @tsaucier-sf

Reply:
I support this

Reply:
Updated aws provider version.